azure/code/rapids_csp_azure.py

```diff
@@ -110,10 +110,14 @@ def load_hyperparams(self, model_name = 'XGBoost'):
                 'gamma': 0.,
                 'lambda': 1.,
                 'alpha': 0.,
+                'objective':'binary:logistic',
                 'random_state' : 0
             }
             if 'GPU' in self.compute_type:
                 model_params['tree_method'] = 'gpu_hist'
+            else:
+                model_params.update['tree_method'] = 'hist'
```

Comment:
This should be .update(tree_method='hist') or .update({'tree_method': 'hist'}),

Reply:
This was old code, will delete, adding histogram in `fit_xgboost` : model_params.update({'tree_method': 'hist'})